recipe/meta.yaml

```diff
@@ -37,8 +37,8 @@ outputs:
         - wheel
```

Comment:
`flit_core` should be used here. See [this](https://github.com/psf/cachecontrol/blob/v0.14.0/pyproject.toml#L2)

Reply:
I'm not sure in what cases flit_core should be used instead of, or alongside, pip, setuptools and wheel. @cbouss do you know about this?

Reply:
Because that's the build system used by controlcache, see https://github.com/psf/cachecontrol/blob/v0.14.0/pyproject.toml#L1-L3.

Reply:
`flit-core` should be used. setuptools and wheel are not required.
We have to follow the `[build-system]` section in `pyproject.toml`. 
Wheel is only required with setuptools (as other backend are self-sufficient for wheel creation).

Reply:
>Wheel is only required with setuptools (as other backend are self-sufficient for wheel creation).
got it